### ISSUE TITLE
Net-Protocol: clientSignOn / clientSignOff objects

### ISSUE THREAD

**Issue by ANDERSONTECHWERKS** [ANDERSONTECHWERKS/Transponder#5]
Idea: send clientSignOn and clientSignOff objects to signal the start / stop of the payload request between tClients and tServers!

```
- Requires careful management of the order in which input/output streams are established between tClients/tServers
- After each stream is established - transmit the payload until stopFlag or clientSignOff is received
- createIOStreams() in tServer will need split into createInputStream()/createOutputStream() in order to first establish the inputStream for clientSignOn, then transmit payload via outputStream, while keeping inputStream watching for clientSignOff.
```

Check out: https://stackoverflow.com/questions/14110986/new-objectinputstream-blocks/14111047
...In case you need reminded why your IOstreams are funny

TL;DR: They need to be "handshook" manually.

**Comment by ANDERSONTECHWERKS** [comment]
IOStreams have been successfully implemented.
